frontend/src/components/TextField/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import styles from "./style.module.css";
+
+interface TextFieldProps {
+  label: string;
+  textValue: string;
+  size?: "big" | "small";
+  // eslint-disable-next-line no-unused-vars
```

Comment:
This issue is caused by ESlint being a dummy & was fixed in a2d2f6b by adding the following to `frontend/.eslintrc.json`:

```
"no-unused-vars": "off",
"@typescript-eslint/no-unused-vars": ["error"],
```

I recommend removing this line by merging those changes from a2d2f6b

Reply:
Oh yep, totally forgot to remedy this after merging in the changes